extension/llm/runner/stats.h

```diff
@@ -82,8 +82,6 @@ struct ET_EXPERIMENTAL Stats {
   long aggregate_sampling_timer_start_timestamp = 0;
 };
 
-static constexpr auto kTopp = 0.9f;
```

Comment:
isn't this bc-breaking because this header doesn't include sampler.h?

Reply:
This constant belong to `sampler.h` not `stats.h`. Anyone using this constant should already imported `sampler.h` so it's not breaking anything. They don't have to include `stats.h` right now if they just need to use this constant.